model/user/user.go

```diff
@@ -323,6 +323,44 @@ func (u *DBUser) IncReauthAttempts() error {
 	return nil
 }
 
+func (u *DBUser) DeleteAllRoles() error {
```

Comment:
Is this ever useful, given that there are some minimal roles that all users have?

Reply:
Yep this will likely be used in offboarding